<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findData</class>
 <widget class="QMainWindow" name="findData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#findData
{
	background-image: url(:/images/find.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="find">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>481</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
image: url(:/images/find.png); 
background-color: transparent;
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: transparent;
font: bold 22px;
min-width:4m;
padding: 6px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdSource">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Source</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbProtocolo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Protocolo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>200</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbDestino">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Destino</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
